pkg/nodeconfig/nodeconfig.go

```diff
@@ -149,6 +149,10 @@ func (nc *nodeConfig) Configure() error {
 		return err
 	}
 
+	if err := nc.Windows.DeconfigureWICD(); err != nil {
+		return errors.Wrap(err, "deconfiguring WICD failed")
```

Comment:
IMO `return err` is sufficient, since DeconfigureWICD itself is a wrapper func that should give a descriptive enough err